Migrate to @babel/parser

skyiea opened the pull request:
Migration https://github.com/wix/okidoc/issues/70

Also contains:
- fix wallaby config
- add test for https://github.com/wix/okidoc/issues/59

bodia-uz commented:
Could we split it into few PR?

bodia-uz commented on packages/okidoc-md/src/utils/parseSource.js:
```diff
@@ -1,15 +1,21 @@
     return parse(source, {
       sourceType: 'module',
       plugins: [
+        'jsx',
```
lets move this commit to separate PR

skyiea commented:
> Could we split it into few PR?
> 
> * wallaby
> * babel parser
> * ..

What's the profit?
Those are tiny fixes, split by commits (no need to squash).
